src/battle/item/coconut/coconut.c

```diff
@@ -31,14 +31,14 @@ Gfx N(coconutDL)[] = {
     gsSPEndDisplayList(),
 };
 
-s32 D_802A1B50_7304B0[] = {
+s32 N(coconutEntity)[] = {
```

Comment:
Not sure I like the name? Maybe coconutIconScript/coconutIconCommandList or something?

Reply:
An 'icon script' is a different thing, and a 'command list' is really vague imo. We need to decide what to call these scripts anyway - 'entity animation' may work?